packages/create-app/templates/default-app/packages/app/src/components/Root/Root.tsx

```diff
@@ -65,7 +74,47 @@ export const Root = ({ children }: PropsWithChildren<{}>) => (
       <SidebarDivider />
       <SidebarGroup label="Menu" icon={<MenuIcon />}>
         {/* Global nav, not org-specific */}
-        <SidebarItem icon={HomeIcon} to="catalog" text="Home" />
+        <SidebarItem icon={HomeIcon} to="catalog" text="Home">
```

Comment:
Tbh we're leaning towards not shipping this in `create-app`, since it competes quite a bit with the homepage and search as the better way to look things up. Perhaps there's some section in the documentation where it could make sense to add this as a possible pattern, or perhaps in contrib? Think we could ship this in the example app either way, just as an experiment.

Reply:
Yup, totally cool with that, sort of thought that might be how this ended up. I'll probably get the changes in place later this week then

Reply:
Alright, I've removed this from `create-app` and added it to the docs instead. I also removed the changeset seeing as I don't think it's needed anymore.